Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/IPropertyEditor.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#include <AzToolsFramework/UI/PropertyEditor/InstanceDataHierarchy.h>
+#include <QtCore/QSet>
+#include <AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.hxx>
```

Comment:
Can remove this include and instead just do a forward declare for `PropertyRowWidget`.

I imagine the methods that use `PropertyRowWidget` will end up staying RPE specific, but we'll need to investigate more to determine for sure.

Reply:
Trying a forward declare for the PropertyRowWidget causes a crash, I left it in for now but definitely will investigate more once I look into all the RPE methods that were removed